README.md

```diff
@@ -14,37 +14,113 @@ All bugs, stories, and backlog for this project are managed through [Hyperledger
 in project IS (note that regular Indy tickets are in the INDY project instead...). Also, join
 us on [Hyperledger's Rocket.Chat](https://chat.hyperledger.org/) at #indy-sdk to discuss.
 
-## Building Indy SDK
+
+## How to install
+### Release channels
+May be one of these values: 
```

Comment:
We are talking here about release channels, not about values that will be used as patterns.